batect.yml

```diff
@@ -50,6 +50,7 @@ containers:
     build_args:
       USER_ID: <{docker_build_user_id}
     build_directory: <{batect.project_directory}
+    dockerfile: Dockerfile-legacy
```

Comment:
Nit: Maybe make this option go first to not split the "build"-type options and to resemble `docker run` CLI argument order a bit.